src/stories/Blocks/status-userprofile/statusUserprofile.tsx

```diff
@@ -0,0 +1,63 @@
+import ListEmpty from "../../Library/Lists/list-empty/ListEmpty";
+import {
+  ListDashboard,
+  ListDashboardProps,
+} from "../../Library/Lists/list-dashboard/ListDashboard";
+import {
+  LinkFilters,
+  LinkFilter,
+} from "../../Library/link-filters/LinkFilters";
+
+export type StatusUserprofileProps = {
+  loansTitle: LinkFilter[];
+  loans: ListDashboardProps[];
+  reservationsTitle: LinkFilter[];
+  reservations: ListDashboardProps[];
+};
+
+export const StatusUserprofile = (props: StatusUserprofileProps) => {
+  const { loansTitle, loans, reservationsTitle, reservations } = props;
+  return (
+    <div className="status-userprofile">
+      <div className="status-userprofile__column">
+        <LinkFilters filters={loansTitle} />
+
+        {loans.length ? (
+          loans.map((item, index) => (
+            <div key={index} className="status-userprofile__loans-list-item">
```

Comment:
Won't react complain over this in the browser console? I think I remember getting an error in the browser when I used index as the key for mapped elements. 
Funnily enough, I actually JUST made a function to generate IDs out of indexes for exactly this purpose, but it's in my current open, yet un-merged PR (#113).

Reply:
It doesn't seem to be the case with errors in the console. But thanks for the heads up.